### PR TITLE
adds feature blocks

### PR DESCRIPTION
**Pull request by leejordan**
I've standardised some of the feature blocks used on the shop and improved their flexibility in terms of layout options and art direction for photography.

pistachio-features

### CODE REVIEW

File: less/modules/features.less [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    text-shadow: 1px 1px 3px @text-colour;
+}
+
+.feature__footer {
+    .transition(background-color 0.05s ease-out);
+    display: block;
+    width: 100%;
+    margin-top: auto;
+    margin-bottom: 0;
+    padding: 0.5em 1em;
```
Comment (notlee) [inline less/modules/features.less]: Should we be sticking to variables to define paddings/margins consistently?

File: less/modules/features.less [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    @media (max-width: @screen-lg-min) {
+        background-position: 70%;
+    }
+
+    @media (max-width: @screen-md-min) {
+        min-height: 100px;
+        background-size: 150%;
+        background-position: top right;
+
+        .feature__side {
```
Comment (notlee) [inline less/modules/features.less]: This classname doesn't feel very intuitive. Would something like `.feature__content` make more sense?

**Comment by notlee** [comment]
@leejordan Awesome! 💃 

Found an issue with text not wrapping IE10:
screen shot 2016-06-24 at 11 59 19

And there might be an annoying rendering issue in IOS potentially which may or may not be worth looking at:
screen shot 2016-06-24 at 12 05 56

**Comment by leejordan** [comment]
@notlee I have addressed your comments apart form the ios bug which is proving tricky to fix so I'll leave it for another day as it's not a breaking bug

**Pull request merged**